mithril-aggregator/Cargo.toml

```diff
@@ -19,6 +19,7 @@ config = "0.13.4"
 flate2 = "1.0.28"
 hex = "0.4.3"
 mithril-common = { path = "../mithril-common", features = ["full"] }
+mithril-doc-derive = { path = "../mithril-doc-derive" }
```

Comment:
From what's see the common practice is to re-export the "derive crate" types in its parent type so the crates that use them don't have to import two crates (ie: [serde](https://github.com/serde-rs/serde/blob/master/serde/src/lib.rs#L330) or [clap](https://github.com/clap-rs/clap/blob/master/src/lib.rs#L104)).
Can something like this be done here ?